src/gerber/convert-soup-to-gerber-commands/index.ts

```diff
@@ -160,7 +160,7 @@ export const convertSoupToGerberCommands = (
 
           let initialX = element.anchor_position.x
           let initialY = element.anchor_position.y
-          const fontSize = element.font_size
+          const fontSize = element.font_size * 0.7 // mm
```

Comment:
This seems jank no? Dont we want the font size to be accurate? Am i missing something?